projects/akita-ng-fire/src/lib/collection/collection.service.ts

```diff
@@ -393,27 +392,27 @@ export class CollectionService<S extends EntityState<any, string>>  {
 
 
   /** Return the current value of the path from Firestore */
-  public async getValue(options?: Partial<SyncOptions>): Promise<getEntityType<S>[]>;
-  public async getValue(ids?: string[], options?: Partial<SyncOptions>): Promise<getEntityType<S>[]>;
+  public async getValue(options?: Partial<SyncOptions>): Promise<EntityType[]>;
+  public async getValue(ids?: string[], options?: Partial<SyncOptions>): Promise<EntityType[]>;
   // tslint:disable-next-line: unified-signatures
-  public async getValue(query?: QueryFn, options?: Partial<SyncOptions>): Promise<getEntityType<S>[]>;
-  public async getValue(id?: string, options?: Partial<SyncOptions>): Promise<getEntityType<S>>;
+  public async getValue(query?: QueryFn, options?: Partial<SyncOptions>): Promise<EntityType[]>;
+  public async getValue(id?: string, options?: Partial<SyncOptions>): Promise<EntityType>;
```

Comment:
guess `id` shouldn't be optional on this line?

Reply:
indeed